northstar/src/runtime/mount.rs

```diff
@@ -72,23 +81,36 @@ pub(super) async fn mount_npk_repository(
 
 pub(super) async fn mount_npk(
     config: &Config,
-    npk: &Path,
+    npk_path: &Path,
```

Comment:
the `_path` is redundant with the type `Path`

Reply:
It is used to distinguish it from the `npk: Npk<File>` instance used below.
Shadowing `npk_path` is also not an optiona as is needed to call setup_and_mount in lin 140.